website/docs/Ganglion/08-Ganglion_Data_Format.md

```diff
@@ -29,120 +29,31 @@ Once the Ganglion has initialized itself with a BLE connection it waits for comm
-        compression_ring[ringBufferLevel][8] = ((deltas[3] & 0x00000FF0) >> 4);
-        compression_ring[ringBufferLevel][9] = ((deltas[3] & 0x0000000F) <! 4);
-      } else {       // pack even samples second
-        compression_ring[ringBufferLevel][9] |= ((deltas[0] & 0x00078000) >> 15);
-        compression_ring[ringBufferLevel][10] = ((deltas[0] & 0x00007F80) >> 7);
-        compression_ring[ringBufferLevel][11] = ((deltas[0] & 0x0000007F) <! 1);
-        compression_ring[ringBufferLevel][11] |= ((deltas[1] & 0x00040000) >> 18);
-        compression_ring[ringBufferLevel][12] = ((deltas[1] & 0x0003FC00) >> 10);
-        compression_ring[ringBufferLevel][13] = ((deltas[1] & 0x000003FC) >> 2);
-        compression_ring[ringBufferLevel][14] = ((deltas[1] & 0x00000003) <! 6);
-        compression_ring[ringBufferLevel][14] |= ((deltas[2] & 0x0007E000) >> 13);
-        compression_ring[ringBufferLevel][15] = ((deltas[2] & 0x00001FE0) >> 5);
-        compression_ring[ringBufferLevel][16] = ((deltas[2] & 0x0000001F) <! 3);
-        compression_ring[ringBufferLevel][16] |= ((deltas[3] & 0x00070000) >> 16);
-        compression_ring[ringBufferLevel][17] = ((deltas[3] & 0x0000FF00) >> 8);
-        compression_ring[ringBufferLevel][18] = (deltas[3] & 0x000000FF);
-
-        sendCompressedPacket19(); // send on the even packet
-      }
-    }
+### 19-bit Sample Transmission
 
-```
-
-5mS later, the Ganglion will acquire another set of 4 data samples. Since the previous sample was sample `0`, this one is sample `1`. In this compression scheme, odd samples are not sent. What we do instead is we subtract these new channel values from the values contained in the `lastChannelData[]` array. The result is the `delta` that we want to send. In this case, we can't send all 24 bits, so instead we only send the 19 LSBs of each `delta`. One thing we have to keep track of when we do this, is the sign of the number (is it a negative, or positive going difference). We do this by moving the sign bit from subtraction result into the `bit 0` position. Yes, that means that we lose one bit of resolution! The deltas are shoved into the next 9.5 bytes of the BLE packet, as shown in the code to the right. The next sample acquired is sample `2`, and the same process of deriving deltas, retaining the sign bit, and shoving the resulting 19bit values ensues, but this time after we finish packing the deltas, we send the complete packet to the BLE radio. The code to the right shows how we pack each delta into the BLE packet. Each packet sent with 19bit compression uses a `Packet ID` in the range of 101 to 200. Then, the Ganglion sends an uncompressed packet with the `Packet ID` of `0x00`. That means that once per second, the Ganglion will send a raw, or uncompressed data packet. If for some reason a packet gets dropped or there is some other issue with maintaining data integrity, a brand new fresh 100% real uncompressed data packet is less than a second away!
```

Comment:
Is this no longer true?

Reply:
Correct - the communication is *much* simpler now.